app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+
+    if len(split_command) != 3 or split_command[0] != "cp":
+        print("Invalid command")
```

Comment:
Consider providing more detailed feedback to the user about why the command is invalid. For example, you could specify whether it's due to incorrect command syntax or missing arguments.